cli/utils.py

```diff
@@ -23,3 +32,48 @@ def main():
 def download_model_artifact(github_token, run_number, output_dir):
     """Download model output from github action publish and deploy workflow. """
     github_utils.download_model_artifact(github_token, output_dir, run_number=run_number)
+
+
+@main.command()
+@click.option(
+    "--csv-path-format",
+    default="combined-{git_branch}-{git_sha}-{timestamp}.csv",
+    show_default=True,
+    help="Filename template where CSV is written",
+)
+def save_combined_csv(csv_path_format):
+    """Save the combined datasets DataFrame, cleaned up for easier comparisons."""
+    csv_path = csv_path_format.format(
+        git_sha=subprocess.check_output(
+            ["git", "describe", "--dirty", "--always", "--long"], text=True
+        ).strip(),
+        git_branch=subprocess.check_output(
+            ["git", "symbolic-ref", "--short", "HEAD"], text=True
+        ).strip(),
+        timestamp=datetime.now().strftime("%Y%m%dT%H%M%S"),
+    )
+
+    timeseries = combined_datasets.build_us_timeseries_with_all_fields()
+    timeseries_data = timeseries.data
+
+    timeseries_data = cleanup_df(timeseries_data)
+    timeseries_data.info()
+
+    write_df_as_csv(timeseries_data, csv_path, structlog.get_logger())
+
+
+def cleanup_df(df):
```

Comment:
I'd love to write a couple quick unit tests on this function!

Reply:
https://github.com/covid-projections/covid-data-public/pull/62 has tests.